plugins/nginx-vhosts/commands

```diff
@@ -180,18 +28,52 @@ EOF
     fi
 
     tail "$NGINX_LOGS_ARGS" "$NGINX_LOGS_PATH"
-    ;;
+  ;;
+
+  nginx:disable)
+    [[ -z $2 ]] && echo "Please specify an app to run the command on" && exit 1
+    verify_app_name "$2"
+    APP="$2"
+
+    eval "$(config_export app $APP)"
+    if [[ -z "$NO_VHOST" ]]; then
+      config_set --no-restart $APP NO_NGINX=1
```

Comment:
We should probably prefix this with `DOKKU_`. Any config var that is dokku specific will then be less likely to conflict with a user env var...